templates/cloud_welcome_email.html

```diff
@@ -1,116 +1,169 @@
-                {{.Props.AddAppsSubInfo}} <a style="text-decoration: none; color: #6DA4EB;" target="_blank" rel="noopener noreferrer" href="{{.Props.AppMarketPlaceLink}}">{{.Props.AppMarketPlace}}</a>
-            </p>
-        </div>
+                                </td>
+                            </tr>
+                        </table>
+                        <table align="center" border="0" cellpadding="0" cellspacing="0">
+                            <tr>
+                                <td style="padding: 20px 20px 10px; text-align:center;">
+                                    <img src="{{.Props.WorkSpacePath}}/static/images/invite_illustration.png"
+                                        width="320px" alt="invite_illustration">
 
-        <div style="position: relative; margin-left: -15px; margin-bottom: 50px;">
-            <span>
-                <svg width="15" height="15" viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-                    <path
-                        d="M11.84 14.0737H0.16V15.7337H11.84V14.0737ZM5.16 0.73373H6.84V9.23373L10.58 5.49373L11.76 6.65373L6 12.4137L0.26 6.65373L1.42 5.49373L5.16 9.23373V0.73373Z"
-                        fill="#3D3C40" />
-                </svg>
-            </span>
-            <span
-                style="position: static; width: 492px; height: 30px; left: 5px; top: 36px; font-family: Open Sans; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40; flex: none; order: 0; flex-grow: 0; margin: 0px 5px;">
-                {{.Props.DownloadMMInfo}}
-            </span>
-            <p
-                style="position: static; width: 458px; height: 20px; left: 0px; top: 0px; font-family: Open Sans; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40; flex: none; order: 1; flex-grow: 0; margin: 0px 0px; margin-left: 24px;">
-                {{.Props.SignInSubInfo}} <a style="text-decoration: none; color: #6DA4EB;" target="_blank" rel="noopener noreferrer" href="{{.Props.DownloadMMAppsLink}}">{{.Props.MMApps}}</a> {{.Props.SignInSubInfo2}}
-            </p>
-        </div>
+                                </td>
+                            </tr>
+                        </table>
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                            <tr>
+                                <td>
+                                    <p
+                                        style="font-family: Open Sans; font-style: normal; font-weight: 400; font-size: 12px; line-height: 16px; text-align: center; color: rgba(61, 60, 64, 0.56);">
+                                        © 2020 Mattermost, Inc. 855 El Camino Real, 13A-168, Palo Alto, CA, 94301
```

Comment:
I think there's a translation for this (if not, let's add one). I know it's probably language agnostic but if we need to change it in the future we only have to change once